examples/factors/README.md

```diff
@@ -2,21 +2,202 @@
+To see this example in action, [install Rust](install-rust) and use `cargo run` in this directory to run it:
+```
+cargo run
+```
 
-Choosing a simple example necessarily excludes all of the more complex use cases -- so if you are looking for anything beyond the basics, we recommend looking at other examples in this repository!
+# Tutorial
+## How to Recreate _Factors_
 
-## Run this example
+This example was chosen because it is straightforward and is a good introduction for beginners. If you're looking to get started creating RISC Zero zkVM projects, you're in the right place! We'll spend the rest of this readme walking you through how to recreate the factors example for yourself, assuming no prior knowledge of RISC Zero.
 
-First, make sure [rustup](https://rustup.rs) is installed. This project uses a [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html). The [`rust-toolchain`](rust-toolchain) file will be used by `cargo` to automatically install the correct version.
+## Step 1: Create a new project
+
+First, [install rust](install-rust) if you don't already have it. Next you can create a RISC Zero zkVM project with boilerplate already filled out using our [`cargo risczero` tool](cargo-risczero):
+```
+## Install from crates.io
+cargo install cargo-risczero
 
-To build all methods and execute the method within the zkVM, run the following command:
+## Navigate to where you want to create your project
+cd wherever/you/want
 
+## Create a project from our starter template
+cargo risczero new factors
 ```
-cargo run --bin factors
+This will create a project named `factors` in the directory where you ran the `cargo risczero new` command. Now we can enter our new project's directory and start working on it!
+```
+cd factors
+```
+
+## Step 2: Give the guest program a name
+
+This project will call a program that executes on the guest zkVM.
+It's currently named `method_name`.
+We want to name it something that represents what the guest program does -- let's call it `multiply`.
+Edit `methods/guest/Cargo.toml`, changing the line `name = "method_name"` to instead read
 ```
+name = "multiply"
+```
+
+In order to access this guest code from the host driver program, the host program `factors/host/src/main.rs` includes two guest methods:
+
+```rust
+use methods::{METHOD_NAME_ELF, METHOD_NAME_ID};
+```
+
+Both of these must be changed to reflect the new guest program name:
+```rust
+use methods::{MULTIPLY_ELF, MULTIPLY_ID};
+```
+(As an aside, if you add more than one callable guest program to your next RISC Zero zkVM project, you'll need to include these `ELF` and `ID` references once for each guest file.)
```

Comment:
Might be worth pointing to an example that uses two guest programs to make this more explicit

Reply:
It looks like our only example with multiple guest programs is Voting Machine, so updating for this depends on https://github.com/risc0/risc0/pull/513 (or creating a multi-program feature example -- which would be nice but is probably even further off)